lib/android/app/src/test/java/com/reactnativenavigation/viewcontrollers/navigator/NavigatorTest.java

```diff
@@ -359,18 +359,6 @@ public void onSuccess(String childId) {
         });
     }
 
-    @Test
```

Comment:
Doesn't look like we need to delete this test

Reply:
this checks creation of stack to make sure that set-root works which i replaced in the stack test